.github/workflows/solvcon_install_devenv.yml

```diff
@@ -0,0 +1,90 @@
+name: solvcon_install_devenv
+
+on:
+  schedule:
+    - cron: '21 18 * * 5'
```

Comment:
This weekly job is expected to run on every Friday so we may check if it is triggered accordingly on Friday.